### DISCUSSION TITLE
My link isn't working

### DISCUSSION THREAD

**Discussion by Johannefotso** [community/community#60724] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hey guys I need some help. I just publish my page on GitHub but my link for my resume isn't working but it was woking before.
https://johannefotso.github.io/Portfolio/
any help will be appreciate.

**Comment by Singulariity** [comment]
This link won't work.
```
/Users/Jojo/Desktop/important/Fotso Johanne Resume.docx.pdf
```

You have to do like this:
```
https://raw.githubusercontent.com/Johannefotso/<repository>/<branch>/<file path>
```
